docs/calico-with-docker/docker-network-plugin/ManualSetup.md

```diff
@@ -42,8 +42,8 @@ up a cluster.
 Follow the instructions for installing [Docker][docker].  A version of 1.9 or
 greater is required.
 
-To use the multi-host native networking feature of Docker, the Docker daemon
-needs to be run specifying a cluster store.  If using etcd as a cluster store,
+To use Calico as a libnetwork plugin, the Docker daemon
```

Comment:
Our docs describe this as a "Docker network plugin" - so we should probably stick with that.  I'm loathe to talk about libnetwork plugin because to me that seems like an implementation detail for the network plugin provider.